pkg/service/dependencies.go

```diff
@@ -21,13 +23,16 @@ import (
 type Dependencies struct {
 	Storage     base.Storage
 	Manipulator Manipulator
+	MetricService metrics.MetricService
 }
 
 // NewDependencies constructs new Dependencies based on the config.DataSource().Kind
 // Currently, it supports only one Manipulator
-func NewDependencies() (*Dependencies, error) {
+func NewDependencies(registry *prometheus.Registry) (*Dependencies, error) {
 	s := config.DataSource()
-	deps := &Dependencies{Manipulator: NewManipulator(native.NewBildProcessor(), getDefaultParams())}
+	metricService := metrics.NewPrometheus(registry)
```

Comment:
We should have a way to switch metrics service between `prometheus` and `statsd`, similar to storage backend.